core/dbt/task/deps.py

```diff
@@ -252,8 +255,12 @@ def nice_version_name(self):
         return "revision {}".format(self.version_name())
 
     def incorporate(self, other):
+        # if one is True, make both be True.
+        warn_unpinned = self.warn_unpinned and other.warn_unpinned
```

Comment:
What's the intended logic here? This comment sounds to me like `self.warn_unpinned or other.warn_unpinned`, right?

Reply:
Whoops! My thought was that it should be False if either was False since it means that at least someone knew what they were doing and didn't want to be bothered. I'm not sure why I wrote `True` twice there in that comment.